### PR TITLE
Initial checkin for SDK readme

### PR DESCRIPTION
**Pull request by lostintangent**
We can expand on this later, but this provides the basic getting started and reference docs for users wanting to take advantage of the new Node.js management SDK.

### CODE REVIEW

**Comment by silhouettes** [comment]
There are a few places where it should be 'retrieves' instead of 'retreives'

File: sdk/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# Node.js SDK for CodePush
+
+A JavaScript library for programatically managing your CodePush account (e.g. creating apps, promoting releases), which allows authoring Node.js-based build and/or deployment scripts, without needing to shell out to the [CLI](https://github.com/Microsoft/code-push/blob/master/cli/README.md).
```
Comment (silhouettes) [inline sdk/README.md]: programmatically

File: sdk/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+- __getCollaborators(appName: string): Promise&lt;CollaboratorMap&gt;__ - Retrieves the list of collaborators associated with the specified app.
+
+- __getDeployment(appName: string, deploymentName: string): Promise&lt;Deployment&gt;__ - Retrieves the metadata for the specified app deployment.
+
+- __getDeploymentHistory(appName: string, deploymentName: string): Promise&lt;Package[]&gt;__ - Retreives the list of releases that have been made to the specified app deployment.
+
+- __getDeploymentMetrics(appName: string, deploymentName): Promise&lt;DeploymentMetrics&gt;__ - Retreives the installation metrics for the specified app deployment. 
+
+- __getDeployments(appName: string): Promose&lt;Deployment[]&gt;__ - Retrieves the list of deployments associated with the specified app.
```
Comment (silhouettes) [inline sdk/README.md]: Promise

File: sdk/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+- __removeDeployment(appName: string, deploymentName: string): Promise&lt;void&gt;__ - Removes the specified deployment from the specified app.
+
+- __renameApp(oldAppName: string, newAppName: string): Promise&lt;void&gt;__ - Renames an existing app.
+
+- __renameDeployment(appName: string, oldDeploymentName: string, newDeploymentName: string): Promise&lt;void&gt;__ - Renames an existing deployment within the specified app.
+
+- __rollback(appName: string, deploymentName: string, targetRelease?: string): Promise&lt;void&gt;__ - Rolls back the latest release within the specified deployment. Optionally allows you to target a specific release in the deployment's history, as opposed to rolling to the previous release.
+
+- __transferApp(appName: string, email: string): Promise&lt;void&gt;__ - Transfers the ownership of the specified app to the specified account.
```
Comment (shishirx34) [inline sdk/README.md]: Do you think we should add ```clearDeployment``` in here as well, or wait for the changes to go through?

Comment (lostintangent) [inline sdk/README.md]: This is simply documenting what is already in master, as well as what the production server accommodates. We can update the readme as new features get rolled in.

**Comment by silhouettes** [comment]
Perfecto!

File: README.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
...
 
-This repo includes the [management CLI](http://microsoft.github.io/code-push/docs/cli.html) as well as some shared code that is used by both the Cordova and React Native client SDKs. To get started using CodePush, refer to our [documentation](http://microsoft.github.io/code-push/index.html#getting_started), otherwise, read the following steps if you'd like to build/contribute to the project from source.
+This repo includes the [management CLI](https://github.com/Microsoft/code-push/tree/master/cli) and [Node.js management SDK](https://github.com/Microsoft/code-push/tree/master/sdk), which allow you to manage and automate the needs of your Cordova and React Native apps. To get started using CodePush, refer to our [documentation](http://microsoft.github.io/code-push/index.html#getting_started), otherwise, read the following steps if you'd like to build/contribute to the project from source.
```
Comment (shishirx34) [inline README.md]: which __allows__ you to...

**Comment by shishirx34** [comment]
LGTM

**Pull request merged**